nameko_slack/rtm.py

```diff
@@ -0,0 +1,109 @@
+import re
+
+import eventlet
+from nameko.extensions import Entrypoint, ProviderCollector, SharedExtension
+from slackclient import SlackClient
+
+
+EVENT_TYPE_MESSAGE = 'message'
+
+
+class SlackRTMClient(SharedExtension, ProviderCollector):
```

Comment:
Abbreviations in capitals look somehow better to me. Many libraries use it that way, `SQLAlchemy`, `SSLError` of `requests` and `urllib3`, ...)